packages/kit/test/apps/basics/test/test.js

```diff
@@ -1627,9 +1627,21 @@ test.describe.parallel('Routing', () => {
 			let requests = [];
 			page.on('request', (r) => requests.push(r.url()));
 
-			await app.prefetch('/routing/prefetched');
-			expect(requests.length).toBe(2);
-			expect(requests[1]).toBe(`${baseURL}/routing/prefetched.json`);
+			// also wait for network processing to complete, see
+			// https://playwright.dev/docs/network#network-events
+			await Promise.all([
+				app.prefetch('/routing/prefetched'),
+				page.waitForResponse(`${baseURL}/routing/prefetched.json`)
```

Comment:
Micro nitpick: I think we should flip these two lines, according to playwright's convention.

Reply:
Done!